### DISCUSSION TITLE
GSoC'24: OpenVINO Messenger AI-Assistant for an AI P

### DISCUSSION THREAD

**Discussion by rghvsh** [openvinotoolkit/openvino#23640] [Google Summer of Code]
Hi @adrianboguszewski ! My name is Raghav Sharma, I would like to reach out to the mentors for: 19. OpenVINO Messenger AI-Assistant for an AI PC.

Thank You 
Raghav Sharma

**Comment by rkazants** [comment]
@DimaPastushenkov, please take a look.

**Comment by DimaPastushenkov** [comment]
Hi @rghvsh, please feel free to reach out to me per email. My address you can find in my github profile. Thanks! Looking forward to our further conversations.

**Comment by rghvsh** [comment]
Hi @rkazants @DimaPastushenkov, thank you for your reply, will make sure to reach out.

Thank You
Raghav Sharma

**Comment by rghvsh** [comment]
Hi @DimaPastushenkov! I have sent you an email yesterday(writing to make sure it does not got to junk)

Thank You
Raghav
